template-ui/channel-overrides/sikana/components/Card.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <b-link
+    v-on:click="isLeaf? goToContent(node) : false"
+    :to="isLeaf? '' : getTopicUrl(node)"
+    class="text-reset text-decoration-none"
+    v-b-hover="handleHover"
+  >
+    <b-card
+      :overlay="hasTextInImage"
+      :img-src="thumbnail"
+      :img-alt="node.title"
+      class="my-2"
+      :class="{shadow: isHovered}"
+    >
+      <span v-if="!hasTextInImage" class="font-weight-bold">{{ title }}</span>
+    </b-card>
+  </b-link>
+</template>
+
+<script>
+import { getThumbnail, goToContent } from 'kolibri-api';
+import { mapGetters } from 'vuex';
+
+export default {
+  props: ['node'],
+  data() {
+    return {
+      thumbnail: null,
+      isHovered: false,
+    };
+  },
+  computed: {
+    ...mapGetters(['getCardLabel', 'getAsset', 'getLevel']),
+    title() {
+      if (this.node.kind === 'topic') {
+        return this.getCardLabel(this.node);
+      }
+      return this.node.title;
+    },
+    isLeaf() {
+      return this.node.kind !== 'topic';
+    },
+    hasTextInImage() {
+      return this.getLevel(this.node) === 4;
+    }
```

Comment:
Missing trailing comma

```suggestion
    },
```